<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shift Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="locationEdit">
      <property name="placeholderText">
       <string>Enter location</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="startTimeEdit">
      <property name="placeholderText">
       <string>Enter start time (hh:mm)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="endTimeEdit">
      <property name="placeholderText">
       <string>Enter end time (hh:mm)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="employeeSpinBox">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addShiftButton">
      <property name="text">
       <string>Add Shift</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Status messages will appear here.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>